docs/pages/validator-management/vc-configuration.md

```diff
@@ -105,7 +125,7 @@ To start a Lodestar validator run the command:
 
 You should see confirmation that modules have started.
```

Comment:
The sh should stay in here

"```sh"

Reply:
Those are logs, bash syntax highlighting looks wrong

Reply:
Doesn't really benefit the docs if it doesn't look similar to what you would actually even see in your terminal. Just leaving as txt should be fine